dbt/adapters/postgres/relation.py

```diff
@@ -20,19 +20,23 @@
 
 @dataclass(frozen=True, eq=False, repr=False)
 class PostgresRelation(BaseRelation):
-    renameable_relations: FrozenSet[RelationType] = field(default_factory=lambda: frozenset(
-        {
-            RelationType.View,
-            RelationType.Table,
-            RelationType.MaterializedView,
-        }
-    ))
-    replaceable_relations: FrozenSet[RelationType] = field(default_factory=lambda: frozenset(
-        {
-            RelationType.View,
-            RelationType.Table,
-        }
-    ))
+    renameable_relations: FrozenSet[RelationType] = field(
```

Comment:
Accidentally ran linting, apparently that's not running at the moment. The test files were also reformatted below.